internal/exec/shell_utils.go

```diff
@@ -185,13 +185,24 @@ func execTerraformShellCommand(
 		}
 	}()
 
-	// Set the Terraform environment variables to reference the var file
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_plan=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_apply=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_refresh=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_import=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_destroy=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_console=-var-file=%s", varFile))
+	// Define the Terraform commands that may use var-file configuration
+	tfCommands := []string{"plan", "apply", "refresh", "import", "destroy", "console"}
+
+	// Check for existing var-file arguments in TF_CLI environment variables
+	for _, cmd := range tfCommands {
+		envVar := fmt.Sprintf("TF_CLI_ARGS_%s", cmd)
+		existing := os.Getenv(envVar)
+		if existing != "" {
+			u.LogWarning(atmosConfig, fmt.Sprintf("detected '%s' set in the environment; this may interfere with Atmos's control of Terraform.", envVar))
+			// Remove any surrounding quotes from existing value
+			existing = strings.Trim(existing, "\"")
```

Comment:
This seems very odd to me. There should be no wrapping quotes, and if there are wrapping quotes, it's because they were passed explicilty (and escaped). 

e.g.
```
TF_CLI_ARGS_plan='"-var-file=/dev/null"' atmos terraform plan ...
```

Which is the same as
```
TF_CLI_ARGS_plan="\"-var-file=/dev/null\"" atmos terraform plan ...
```

If this is the case, the user is (incorrectly) passing quotes. We shouldn't try to "fix" that, and we should just accept that it's an invalid argument and let Terraform handle the validation.

Reply:
updated!
![CleanShot 2025-01-03 at 14 34 45@2x](https://github.com/user-attachments/assets/bbca99a3-3888-4c9c-bc66-012df6730655)